### ISSUE TITLE
VDC: Derived variables have no fidelity control

### ISSUE THREAD

**Issue by sgpearse** [NCAR/VAPOR#2609]
Derived variables have no fidelity control in VDC datasets

To reproduce:
1) Open a VDC
2) Create a derived variable
3) Render it with any renderer, and the fidelity control cannot be changed from "High"

**Comment by FMInsert** [comment]
when trying to render a variable defined via a python script, I see that various renderers (at least: slice and flow) do not yield more than lowest resolution. While the fidelity button can be pressed and set to 'high', this does not have the desired effect: the 'level of detail' button stays in low resolution (0) and the results are visibly in low resolution (and/or, apparently, simply wrong: the patterns on the slice surface are not just low resolution but pretty weird; the streamlines in the flow have visible corners).

**Comment by clyne** [comment]
I believe that this has been fixed in main. Can you try installing one of the Weekly Builds and see if you are still having trouble?

**Comment by sgpearse** [comment]
Quoted from @FMInsert's replay via email:

> thanks for the suggestion.
> 
> I have now downloaded the most recent Weekly Build (3.6.1, 2022.08.19) and tried to instal it in my recent Mac (Chip M1 Ultra, running on macOS Monterey 12.5.1) --- but it does not open ("VAPOR_3.6.1 app quit unexpectedly").

@clyne - Does someone on the team have an M1 macbook?  The weekly build is working on our intel test machies.

**Comment by clyne** [comment]
Yes, there is one in my office that you are welcome to grab.

**Comment by FMInsert** [comment]
Good news: I have installed the weekly build 3.6.1 in an old Mac machine running on Big Sur. The problem with the python variables is gone ! (I have checked both the slice and flow renderers).

On the other hand, I have double-checked the problem with the M1 Mac machines: I have now tried two different M1 machines (a laptop, additionally to the desktop I was using a few days ago) and the crashes continue.
